quipucords/scanner/ansible/api.py

```diff
@@ -28,10 +36,19 @@ def from_connection_info(  # noqa: PLR0913
         :param username: The username to use for connecting to the server.
         :param password: The password to use for connecting to the server.
         :param ssl_verify: Whether to verify the SSL certificate.
+        :param proxy_url: proxy URL in the format 'host:port'.
         """
         base_uri = f"{protocol}://{host}:{port}"
         auth = HTTPBasicAuth(username=username, password=password)
-        return cls(base_url=base_uri, verify=ssl_verify, auth=auth)
+        session = cls(base_url=base_uri, verify=ssl_verify, auth=auth)
+
+        proxies = {}
+        if proxy_url:
+            proxies[protocol] = "http://" + proxy_url
```

Comment:
same as in the serializer comment: is there a reason so support exclusively HTTP protocol? what if we are dealing with a https proxy?

if that's not the intention, I'd also make the serializer expect the protocol.

Reply:
see comment above